netdata-installer.sh exited with error

<!---
When creating a bug report please:
- Verify first that your issue is not already reported on GitHub
- Test if the latest release and master branch are affected too.
- Provide a clear and concise description of what the bug is in "Bug report 
  summary" section.
- Try to provide as much information about your environment (OS distribution,
  running in container, etc.) as possible to allow us reproduce this bug faster.
- Write which component is affected. We group our components the same way our
  code is structured so basically:
  component name = dir in top level directory of repository
- Describe how you found this bug and how we can reproduce it. Preferable with
  a minimal test-case scenario. You can paste gist.github.com links for larger
  files
- Provide a clear and concise description of what you expected to happen.
-->

##### Bug report summary
netdata-installer.sh exited with error 
cciV04OR.ltrans10.o:(.text+0x3ca3): undefined reference to `LZ4_compress_default'

##### OS / Environment
Linux 3.16.0-9-amd64 #1 SMP Debian 3.16.68-2 (2019-06-17) x86_64 GNU/Linux

##### Netdata version (ouput of `netdata -V`)
Daily build using bash <(curl -Ss https://my-netdata.io/kickstart.sh)
Aug 01 2019

##### Component Name

##### Steps To Reproduce

##### Expected behavior

Hi Paul,

Thanks for the prompt response! Here is the info you requested.
I have attached output from installation.


cat /etc/os-release =>

PRETTY_NAME="SolydXK"
NAME="SolydXK"
ID=solydxk
ANSI_COLOR="1;31"
HOME_URL="http://www.solydxk.com/"
SUPPORT_URL="http://forums.solydxk.com/"
BUG_REPORT_URL="http://forums.solydxk.com/"

uname -a =>
Linux titanium 3.16.0-9-amd64 #1 SMP Debian 3.16.68-2 (2019-06-17)
x86_64 GNU/Linux

Installation gets to this stage with no errors, then =>

  GEN      netdata
/tmp/cciV04OR.ltrans10.ltrans.o: In function `do_flush_pages.isra.1':
cciV04OR.ltrans10.o:(.text+0x3ca3): undefined reference to
`LZ4_compress_default'
collect2: error: ld returned 1 exit status
Makefile:2574: recipe for target 'netdata' failed
make[2]: *** [netdata] Error 1
make[2]: Leaving directory
'/tmp/netdata-kickstart-lIwzGO/netdata-v1.16.0-108-g2fd634f8'
Makefile:2971: recipe for target 'all-recursive' failed
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory
'/tmp/netdata-kickstart-lIwzGO/netdata-v1.16.0-108-g2fd634f8'
Makefile:1610: recipe for target 'all' failed
make: *** [all] Error 2
 FAILED   

 FAILED   

 ABORTED  netdata-installer.sh exited with error 


=======================================================

On Thu, 2019-08-01 at 06:55 -0700, Paul Emm. Katsoulakis wrote:
> Hi @knarf-llennoc, thank you for your report!
> I am sorry to hear you faced such issues.
> 
> Could you share more details on the issue?
> I am going to need the output of cat /etc/os-release from the system.
> Also the output of kickstart execution from the shell, so that i can
> understand what packages were installed and what not.
> 
> Thanks !
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>

--- a/configure.ac
+++ b/configure.ac
@@ -283,7 +283,7 @@ OPTIONAL_UV_LIBS="${UV_LIBS}"
 
 AC_CHECK_LIB(
     [lz4],
-    [LZ4_decompress_safe],
+    [LZ4_compress_default],
     [LZ4_LIBS="-llz4"]
 )
 

